src/app/views/query-response/pivot-items/pivot-items.tsx

```diff
@@ -1,5 +1,5 @@
-import { getTheme, IPivotItemProps, ITheme, PivotItem } from '@fluentui/react';
-import React from 'react';
+import { getTheme, IPivotItemProps, ITheme, PivotItem, Spinner, SpinnerSize } from '@fluentui/react';
```

Comment:
These are repeated heavily, we can have a `SuspenseLoader` component that we can use to wrap all the suspense content